### DISCUSSION TITLE
lc/1346/

### DISCUSSION THREAD

**Discussion by giscus** [doocs/leetcode#2702] [Announcements]
# lc/1346/

多种编程语言实现 LeetCode、《剑指 Offer（第 2 版）》、《程序员面试金典（第 6 版）》题解

https://leetcode.doocs.org/lc/1346/

**Comment by thinkasany** [comment]
这题md解析有点问题

**Comment by thinkasany** [comment]
<img width="1086" alt="image" src="https://github.com/doocs/leetcode/assets/117748716/0eff77d0-8baf-454e-962d-20717ec496f0">

**Reply by yanglbme** [reply]
有发现这个问题，但是不知道怎么改

**Reply by thinkasany** [reply]
爬下来可以不要```<code>```么，这个会和代码块冲突

**Reply by yanglbme** [reply]
那只能在块内不要 code，其他地方应该还是要的

**Reply by yanglbme** [reply]
我有空改改 docs 分支的渲染吧，加个钩子，原始数据保留着

**Reply by thinkasany** [reply]
以前好像遇到过类似的问题，忘记咋处理来着了，实在不行只能手改去掉了，或者只在某些集合内部做去除code的处理。
